### PR TITLE
Auto-load all stops

### PR DESCRIPTION
**Pull request by akaplo**
I updated the BusInfoBoard's StopSelector page to automatically download and allow users to search through/choose from all of PVTA's stops.  Previously, you had to select a route and then scroll through or type in a stop name.  I made it easier to skip that step (and people don't know that you can type to search for routes/stops, so I made that a little clearer too).

### CODE REVIEW

**Comment by sherson** [comment]
Looks great, thanks!

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.DS_STORE
```
Comment (werebus) [inline .gitignore]: OS or development-environment-specific things should go in your global ignore file

https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

File: StopSelector/index.html [hunk 2]
```diff
@@ -15,6 +15,19 @@
...
+      font-size: 1.5em;
+      margin-before: 0.83em;
+      margin-after: 0.83em;
+      margin-start: 0;
+      margin-end: 0;
+      font-weight: bold; }
+    span {
+      color: "white";
+    }
+  </style>
```
Comment (werebus) [inline StopSelector/index.html]: No inline style, please.  This should go in [the CSS file](https://github.com/umts/BusInfoBoard/blob/master/StopSelector/css/main.css)

Having said that, if you're going to use `<style>` tags, they go inside the `<head>`

File: StopSelector/index.html [hunk 3]
```diff
@@ -38,12 +51,12 @@ <h3 class="text-center or">----Or----</h3>
     </div>
     <div class="row selector-holder">
       <div class="col-md-6">
-        <h2>Filter by routes</h2>
-        <select name="routes[]" class="routes" data-placeholder="Select Routes" multiple></select>
+        <h2>Select Routes</h2> <font color="white">&nbsp;[Optional]</font>
```
Comment (werebus) [inline StopSelector/index.html]: <font> is an [obsolete tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/font). Use CSS instead.  Also, `&nbsp;` should be avoided in general unless you have a very specific reason why

```
Select Routes
[Optional]
```

is unacceptable line-breaking

**Comment by werebus** [comment]
In the future, lets try to get this code in front of a code-review before deploying it.

**Comment by sherson** [comment]
> In the future, lets try to get this code in front of a code-review before deploying it.

Yeah, that was my bad.

File: StopSelector/js/main.js [hunk 4]
```diff
@@ -124,6 +155,41 @@ function stopList(select, stops) {
...
+* throw out everything except one stop.
+* 
+* @param: collection: Object
+* @return: collection: Object <-- the sorted and
+* uniquified version of your input.
+**/
+
+function uniqueifyAndSort(collection){
+  collection = _.uniq(_.union(_.flatten(collection)), _.iteratee('StopId'));
+  collection.sort(function(a,b) {
```
Comment (werebus) [inline StopSelector/js/main.js]: If we're going to have [underscore](http://underscorejs.org/#) in the app, might as well use [`_.sortBy`](http://underscorejs.org/#sortBy)

**Comment by akaplo** [comment]
I've updated some (most) of the things that @werebus mentioned and pushed them to my fork.  I'm never around for our scheduled code reviews but whenever I'm present and you guys have time, doing a code review would be great!

**Comment by werebus** [comment]
So, two non-nit-pick things to mention:

1.  The flow of the page is a little haphazard:  -----Thing1----- OR Thing2 [optional] OR/Now Thing3.  I'm not sure what I would've done differently, though.

2.  One problem with a list of all stops is stuff like this:

**Pull request merged**